src/archiefvernietigingscomponent/destruction/views.py

```diff
@@ -247,6 +252,17 @@ def forms_valid(self, form, inlines):
             ).count(),
         )
 
+        # Check if there are comments from the author
+        if "text" in self.request.POST:
+            comment = DestructionListReviewComment(
+                author=self.request.user, review=destruction_list.last_review()
+            )
+            comment_form = ReviewCommentForm(self.request.POST, instance=comment)
+            if comment_form.is_valid():
+                comment_form.save()
+            else:
+                super().forms_invalid(form, inlines)
```

Comment:
I don't think that putting this code into `forms_valid` method is a good idea in terms of readability. This method is already very fat (and I'm sorry for that). And I don't quite understand why not put the `comment` field into the view form? This view is already inherited from `FormView` so it's already build-in in the view methods
So I think the following steps would improve readability:
* create `DestructionListForm` and move `comment` field there (of just rename `ReviewCommentForm`)
* use DestructionListForm as form_class in `DestructionListDetailView`
* rewrite the form's `save` method and move all this code there 

I'm not sure if @joeribekker agrees with me on this remark though, so I tagged him

Reply:
After discussion on slack, we decided to leave this for now.

This is because the form should only be present when the record manager has `can_update == True`, so to deal with this there would still need to be some complexity in the `get_context` method and in the `forms_valid`.